README.md

```diff
@@ -41,18 +41,82 @@ Returns source map converter from given base64 encoded json string.
 
 Returns source map converter from given base64 encoded json string prefixed with `//# sourceMappingURL=...`.
 
-### fromMapFileComment(comment, mapFileDir)
+### fromMapFileComment(comment, mapFileDir, readMap)
```

Comment:
@prantlf I'm wondering if this even needs `mapFileDir` anymore? Couldn't that just be part of the `readMap` function?

Reply:
I rebased this and took a deeper look and it seems that `mapFileDir` is unnecessary when you allow/require users to provide their own `readMap` function. You can see the path resolution logic in the updated tests.